release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,186 @@
+k6 `v1.0.0-rc1` is here 🎉! This release includes:
+
+- A preview of the next `v1.0.0` release
+- `k6/experimental/webcrypto` promoted to stable and available globally under `crypto`.
+- A revamped end-of-test summary aiming to bring an easier way to understand test results.
+
+## Breaking changes
+
+- [#4541](https://github.com/grafana/k6/pull/4541) Commas(`,`) are now supported in the values of `--tag` cli flag. This is a breaking change as previously a comma meant the start of a new set of tag-values. As comma is a valid symbol for the value of a tag this is necessary to have equivalency between different ways of setting tags. This still allows multiple tags to be set on the cli with multiple `--tag key=value` arguments.
+
+### A revamped end-of-test summary aiming to bring an easier way to understand test results [#4089](https://github.com/grafana/k6/pull/4089)
```

Comment:
@joanlopez do you plan to update the documentation for it here https://grafana.com/docs/k6/latest/results-output/end-of-test?

Reply:
Done here: https://github.com/grafana/k6-docs/pull/1906